src/figma/constants.ts

```diff
@@ -0,0 +1,32 @@
+// There are some extended variables we need to
+// make available internally. This array captures
+// those unique cases.
+//
+// Each string in this array should align with
+// the variable format in Figma:
+//
+// {collection}/{group(s)}/{variable}
+//
+// This information can be found in Figma's
+// Deatils modal when clicking on an extended
+// style by combining the collection information
+// with the variable's full name.
```

Comment:
As an example:

<img width="305" alt="Screenshot 2025-03-12 at 8 58 03 AM" src="https://github.com/user-attachments/assets/f7c9b60b-a2ec-4b52-a0da-b24606234aa1" />

Can copy the collection (bottom) and the full variable name (top)